scripts/gitlab/build_and_test.sh

```diff
@@ -152,3 +156,10 @@ then
         echo "ERROR: failure(s) while running CTest" && exit 1
     fi
 fi
+
+echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+echo "~~~~~ CLEAN UP"
+echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+
+make clean
+rm -rf uberenv_libs
```

Comment:
I think you need to look up a directory for this one?

Reply:
(Resolving because new commit edits address this - uberenv_libs will be in the shared mem folder /dev/shm/ where it'll be deleted anyway)